src/components/common/students/empty-dashboard/empty-dashboard.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+    Dialog,
+    DialogContent,
+    DialogDescription,
+    DialogHeader,
+    DialogTrigger,
+} from "@/components/ui/dialog";
+import { Share } from "@phosphor-icons/react";
+import { MyButton } from "@/components/design-system/button";
+
+export const EmptyDashboard = () => {
+    return (
+        <div
+            className={`flex w-full flex-col items-center justify-center gap-4 rounded-md bg-neutral-50 py-10`}
+            style={{ height: `calc(100vh - 160px)` }}
+        >
+            <img src="/images/svgs/empty-student-dashboard.svg" width={300.8} height={294.7} />

```

Comment:
don't use it like this instead add this image in svgs folder and import from ts file then include it like this and don't include width and height.
< EmptyDashboard/ >